src/uct/tcp/tcp.h

```diff
@@ -74,12 +94,13 @@ typedef struct uct_tcp_ep_ctx {
  */
 struct uct_tcp_ep {
     uct_base_ep_t                 super;
+    uint8_t                       ctx_caps;    /* Which contexts are supported? */
```

Comment:
why `?`

Reply:
this will be needed by the next PR to identify whether EP found in the hash supports RX or TX

Reply:
ok, bt I meant why do you need `?` sign in the end of the comment :)

Reply:
> ok, bt I meant why do you need `?` sign in the end of the comment :)

ah, I see. fixed